src/main/java/seedu/duke/component/VoltageSource.java

```diff
@@ -0,0 +1,19 @@
+package seedu.duke.component;
+
+public class VoltageSource extends Component  {
+    private static final String SI_UNIT = "V";
+
+    public VoltageSource(double value) {
+        super(value);
+    }
+    /**
+     * Returns String constituting value and unit of voltage source of circuit.
+     *
+     * @return String representation of VoltageSource object.
+     */
+
```

Comment:
The JavaDoc should be with its method.
```suggestion

    /**
     * Returns String constituting value and unit of voltage source of circuit.
     *
     * @return String representation of VoltageSource object.
     */
```